ipxedust.go

```diff
@@ -187,7 +195,10 @@ func (c *Server) listenAndServeTFTP(ctx context.Context) error {
 	h := &itftp.Handler{Log: c.Log}
 	ts := tftp.NewServer(h.HandleRead, h.HandleWrite)
 	ts.SetTimeout(c.TFTP.Timeout)
-	c.Log.Info("serving TFTP", "addr", c.TFTP.Addr, "timeout", c.TFTP.Timeout)
+	if c.EnableTFTPSinglePort {
+		ts.EnableSinglePort()
+	}
```

Comment:
Is it possible to check return status of ts.EnableSinglePort ?  what if it fails to enable ?

Reply:
doesn't look like that function has a return value: https://github.com/pin/tftp/blob/0161c5dd2e967493da88cfdf9426b9337afb60ee/server.go#L117

Reply:
from the comment, I see that it impacts performance?

Reply:
yes. Good point. I have added a comment about that. thanks!